src/include/parser/expression/column_value_expression.h

```diff
@@ -0,0 +1,209 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+#include "catalog/catalog_defs.h"
+#include "parser/expression/abstract_expression.h"
+
+namespace terrier::parser {
+
+/**
+ * Represents a column tuple value.
+ */
+class ColumnValueExpression : public AbstractExpression {
+ public:
+  /**
+   * This constructor is called only in postgresparser, setting the column name,
+   * and optionally setting the table name and alias.
+   * Namespace name is always set to empty string, as the postgresparser does not know the namespace name.
+   * Parameter namespace name is included so that the program can differentiate this constructor from
+   * another constructor that sets the namespace name, table name. and column name.
+   * @param namespace_name namespace name
+   * @param table_name table name
+   * @param col_name column name
+   * @param alias alias of the expression
+   */
+  ColumnValueExpression(std::string namespace_name, std::string table_name, std::string col_name, std::string alias)
+      : AbstractExpression(ExpressionType::COLUMN_VALUE, type::TypeId::INVALID, std::move(alias), {}),
+        namespace_name_(std::move(namespace_name)),
+        table_name_(std::move(table_name)),
+        column_name_(std::move(col_name)) {}
+
+  /**
+   * @param table_name table name
+   * @param col_name column name
+   */
+  ColumnValueExpression(std::string table_name, std::string col_name)
```

Comment:
I don't think we need this constructor. This class should always be given a namespace.

Reply:
We need this constructor.  Unqualified object names are allowed because the binder is expected to use a search path of namespaces and bind to the first match.